### DISCUSSION TITLE
PDF forms

### DISCUSSION THREAD

**Discussion by jcos1** [typst/typst#601] [Ideas]
Create pdf forms with fillable fields

- Button
- Radio
- CheckBox
- ListBox
- ComboBox
- Text

[https://docs.rs/pdf_form/latest/pdf_form/enum.FieldType.html](https://docs.rs/pdf_form/latest/pdf_form/enum.FieldType.html)

https://github.com/typst/typst/issues/2421

https://github.com/typst/typst/issues/4368

**Comment by PabloRuizCuevas** [comment]
pdf forms would be great :+1:
